rj_gameplay/rj_gameplay/play/pivot_play.py

```diff
@@ -0,0 +1,53 @@
+import stp.play as play
+import stp.tactic as tactic
+
+from rj_gameplay.tactic import temp_pivot_tactic
+import stp.skill as skill
+import stp.role as role
+from stp.role.assignment.naive import NaiveRoleAssignment
+import stp.rc as rc
+from typing import Dict, Generic, Iterator, List, Optional, Tuple, Type, TypeVar
+import numpy as np
+
+class PivotPlay(play.IPlay):
+    """A play which lines up two robots, one on the right the one on the left
+    """
+
+    def __init__(self):
+        self.target_point  = np.array([1.0,1.0])
+        self.pivot = temp_pivot_tactic.Pivot(self.target_point)
+        self.role_assigner = NaiveRoleAssignment()
+
+
+    def compute_props(self, prev_props):
+        pass
+
+    def tick(
+        self,
+        world_state: rc.WorldState,
+        prev_results: role.assignment.FlatRoleResults,
+        props,
+    ) -> Tuple[Dict[Type[tactic.SkillEntry], List[role.RoleRequest]], List[tactic.SkillEntry]]:
+        # Get role requests from all tactics and put them into a dictionary
+        role_requests: play.RoleRequests = {}
+        if not self.pivot.is_done(world_state):
+            role_requests[self.pivot] = self.pivot.get_requests(world_state, None)
+        else:
+            pass
```

Comment:
Delete the `else: pass`, unless this is a stub, in which case add a TODO above the pass here.

Reply:
(Temp play caveat applies here too though)